packages/connect-core/src/transactions/TransactionRequest.ts

```diff
@@ -21,22 +12,17 @@ export interface Annotation {
 }
 
 export default class TransactionRequest {
-  readonly to!: string
-  readonly from!: string
-  readonly data!: string
-  readonly pretransaction?: {
-    to: string
-    from: string
-    data: string
-  }
-  readonly description?: string
   readonly annotatedDescription?: Annotation[]
-  readonly gas?: string
-  readonly gasPrice?: string
-  readonly gasLimit?: string
-  readonly chainId?: number
+  readonly description?: string
+  readonly data!: string
+  readonly from!: string
+  readonly to!: string
 
   constructor(data: TransactionRequestData) {
-    Object.assign(this, data)
+    this.annotatedDescription = data.annotatedDescription
```

Comment:
Should we use `description` and `descriptionAnnotated`, like we have `transactions` and `transactionDescribed`? That way they will both complete when `description…` is typed, and it makes them a bit easier to identify as being related.

Reply:
Yeah, good point